ui/apps/platform/src/Containers/Clusters/StaticConfigurationSection.js

```diff
@@ -44,9 +44,14 @@ const StaticConfigurationSection = ({
     isManagerTypeNonConfigurable,
     handleChange,
 }) => {
+    const filteredRuntimeOptions = runtimeOptions.filter((option) => {
```

Comment:
This appears to have the desired effect of removing EBPF from selection options, but still displays correctly for secured clusters that are still using eBPF

![Legacy Cluster Creation Page](https://github.com/stackrox/stackrox/assets/1782335/01b1054d-5d47-4f8e-90af-b27f3435b600)
![Existing EBPF Cluster](https://github.com/stackrox/stackrox/assets/1782335/eb35eb81-e641-44f9-a223-b82176811eea)

@pedrottimark - whilst this works, I'm not sure if this is the most appropriate fix/solution, could you please take a look?

Reply:
@Stringy Only one question about edge case of existing secured cluster **with manifest installation method** on older version with EBPF collection method. This change filters out `EBPF` option, so value will appear as **No collection method** but is that reality for collector pods in that secured clusters? This scenario is not expected for customers, correct?

You have my apology that I did not get ahead of this with potential improvement to PatternFly `Select` element in which it is possible to disable individual options :(

@alwayshooin Can you give a second pair of eyes. Summary of changes which I tested locally.
1. Filter `runtimeOptions` array:
    * keep `EBPF` option to display that collection method if needed in existing cluster has that have `isManagerTypeNonConfigurable` like Helm or Operator installation, for which the `Select` element is disabled.
    * skip `EBPF` option so cannot choose for secured cluster with manifest installation method.
3. Replace `runtimeOptions` with `filteredRuntimeOptions` in the only 2 occurrences.

Reply:
i think the code makes sense! maybe just adding a comment to explain the logic?

Reply:
@pedrottimark 
> @Stringy Only one question about edge case of existing secured cluster with manifest installation method on older version with EBPF collection method. This change filters out EBPF option, so value will appear as No collection method but is that reality for collector pods in that secured clusters? This scenario is not expected for customers, correct?

I agree that manifest installation is not expected for customers. I guess we could handle this case by passing the manager type down to this level and adjust the filtering based on that?

@alwayshooin 
> maybe just adding a comment to explain the logic?

Sure! I've expanded the comment to be clearer.